.github/workflows/main.yml

```diff
@@ -0,0 +1,64 @@
+name: Autolens Workspace Test Integration Test
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        #python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
I think we are backwards compatible for 3.6 and 3.7 ?

Reply:
Cool, good to know. I'll enable.